diff --git a/Containerfile b/Containerfile
index 846af1d..98e550b 100644
--- a/Containerfile
+++ b/Containerfile
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG CONTAINER_IMAGE=quay.io/ansible/python-base:latest
+ARG CONTAINER_IMAGE=quay.io/tripleoansible/python-base:stream9
 ARG REMOTE_SOURCE=.
 ARG REMOTE_SOURCE_DIR=/remote-source
 ARG REMOTE_SOURCE_APP_DIR=$REMOTE_SOURCE_DIR
@@ -36,7 +36,7 @@ COPY --from=builder $REMOTE_SOURCE_APP_DIR/scripts/install-from-bindep /output/i
 WORKDIR $REMOTE_SOURCE_APP_DIR
 
 RUN dnf update -y \
-  && dnf install -y python38-wheel git \
+  && dnf install -y python3-wheel git \
   && dnf clean all \
   && rm -rf /var/cache/{dnf,yum} \
   && rm -rf /var/lib/dnf/history.* \
